roles/download/defaults/main.yml

```diff
@@ -1001,7 +1001,7 @@ skopeo_binary_checksum: "{{ skopeo_binary_checksums[image_arch][skopeo_version]
 kube_proxy_image_repo: "{{ kube_image_repo }}/kube-proxy"
 etcd_image_repo: "{{ quay_image_repo }}/coreos/etcd"
 etcd_image_tag: "{{ etcd_version }}"
-flannel_image_repo: "{{ docker_image_repo }}/flannelcni/flannel"
+flannel_image_repo: "{{ docker_image_repo }}/flannel/flannel"
 flannel_image_tag: "{{ flannel_version }}"
 flannel_init_image_repo: "{{ docker_image_repo }}/flannelcni/flannel-cni-plugin"
```

Comment:
HI @ErikJiang 
Should the flannel_init_image_repo change together?

Reply:
It may not be possible to change it because the `v1.2.0` image is not found in [flannel/flannel-cni-plugin](https://hub.docker.com/r/flannel/flannel-cni-plugin/tags?page=1).